hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -428,7 +429,7 @@ genAddressInEra networkId =
   mkVkAddress networkId <$> genVerificationKey
 
 genValue :: Gen Value
-genValue = scale (`div` 10) $ fromLedgerValue <$> arbitrary
+genValue = liftA2 (<>) (pure $ lovelaceToValue $ Lovelace 10_000_000) (scale (`div` 10) $ fromLedgerValue <$> arbitrary)
```

Comment:
Should: not need to divide by `10` anymore and use a bit smaller value, 10ADA is a bit much and I think 1-2 ADA is "more realistic" (I know it was millions of ADA before)